src/main/java/net/geant/nmaas/portal/api/user/UsersController.java

```diff
@@ -615,6 +615,30 @@ public List<UserViewMinimal> searchUser(@RequestParam(required = false) String s
         return result;
     }
 
+    @GetMapping(value = "/users/search/managers", params = {"searchPart"})
+    @PreAuthorize("hasRole('ROLE_SYSTEM_ADMIN') or hasRole('ROLE_DOMAIN_ADMIN') or hasRole('ROLE_VL_MANAGER')")
```

Comment:
I suppose that DOMAIN_ADMIN should not have access to this endpoint since it's dedicated for adding group managers , correct? Also this optional "domainId" parameters can be removed.